src/components/ApplicationForm/Skills.js

```diff
@@ -115,9 +138,8 @@ export default ({ refs, errors, formInputs, onChange, role, handleResume }) => {
 
           {role === 'designer' ? (
             <>
-              <FormRow fieldValue="Personal website/portfolio link">
+              <FormRow fieldValue="Personal website/portfolio link" required>
                 <TextInput
-                  inline
```

Comment:
There's a funky bug - whereby if I type a space into any of these fields it seems to crash the website and become unresponsive 😭, but there isn't a clear log of an error stack either? The error doesn't exist on the long answer question's boxes though